### PR TITLE
Updated Sophie Koonin's talk date and location

### PR DESCRIPTION
**Pull request by yanneves**

### CODE REVIEW

File: _posts/2020-08-20-teaching-code-as-a-foreign-language.md [hunk 1]
```diff
@@ -14,14 +14,9 @@ sponsors:
...
   - coaching
-venue:
-  name: The Skiff
-  link: https://www.theskiff.org/
-  location: https://goo.gl/maps/SPf97i5WkAR2
-  address: Ironworks, 30 Cheapside, Brighton, BN1 4GD
-  latlong: 50.829371,-0.138194
 layout: event.html
 collection: events
+online: true
```
Comment (larister) [inline _posts/2020-08-20-teaching-code-as-a-foreign-language.md:19]: Cool - is this metadata used anywhere?

Comment (yanneves) [reply _posts/2020-08-20-teaching-code-as-a-foreign-language.md:19]: didn't check tbh, just noticed it on the other online events

Comment (yanneves) [reply _posts/2020-08-20-teaching-code-as-a-foreign-language.md:19]: ah yep, it's used in `_includes/partials/eventDetail.html`

Comment (larister) [reply _posts/2020-08-20-teaching-code-as-a-foreign-language.md:19]: Yeah just saw, awesome

**Pull request merged**